configs/paths.go

```diff
@@ -73,19 +87,32 @@ func InitPaths() error {
 		return fmt.Errorf("Failed to init bitrise paths, error: %s", err)
 	}
 
-	inputEnvstorePath, err := filepath.Abs(path.Join(BitriseWorkDirPath, "input_envstore.yml"))
+	// --- Bitrise TOOLS
+	bitriseToolsDirPth := GetBitriseToolsDirPath()
+	if err := pathutil.EnsureDirExist(bitriseToolsDirPth); err != nil {
+		return err
+	}
+	pthWithBitriseTools := os.Getenv("PATH")
+	if !strings.HasPrefix(pthWithBitriseTools, bitriseToolsDirPth) {
```

Comment:
good point, although we'd usually want to enforce that the version we have in `tools/` is the one which will be used, instead one from e.g. `/usr/local/bin`

not sure though whether this should be enforced in the way it is implemented here, as this will not allow you to actually use a different version, e.g. during development, without moving it into `tools/`..